qdrant_docs_mcp/tools/importer.py

```diff
@@ -168,19 +183,28 @@ def extract_from_repo(library: Library, repo: Path, version: str) -> list[Snippe
     """
 
     snippets: list[PartialSnippet] = []
-    for file in chain(
-        repo.glob("**/*.md"), repo.glob("**/*.rst"), repo.glob("**/*.ipynb")
-    ):
+    paths = list(
+        chain(repo.glob("**/*.md"), repo.glob("**/*.rst"), repo.glob("**/*.ipynb"))
+    )
+
+    if source.include_files is not None:
+        filtered_paths: set[Path] = set()
+        for pattern in source.include_files:
+            filtered_paths |= {
+                path for path in paths if path.full_match(repo / pattern)
+            }
+
+        paths = filtered_paths
```

Comment:
Why do we still need `paths` if we have `include_files` with the correct paths? 
Is it possible to do something like `repo.glob(include_files_paths)`?